src/main/java/bio/terra/cli/serialization/userfacing/UFFormatConfig.java

```diff
@@ -0,0 +1,37 @@
+package bio.terra.cli.serialization.userfacing;
+
+import bio.terra.cli.command.shared.options.Format;
+import bio.terra.cli.command.shared.options.Format.FormatOptions;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonPOJOBuilder;
+
+@JsonDeserialize(builder = UFFormatConfig.Builder.class)
+public class UFFormatConfig {
```

Comment:
I don't think you need a separate `UF` class for a single property. 2 reasons:
- The other config properties just print their value (e.g. see the get command for `app-launch` and `browser` properties). This would make `format` inconsistent.
- Having a single "format" key set to the config value doesn't provide any information the caller doesn't already have. They know it's the format because they ran the command `terra config get format`.

Reply:
👍🏻